Update 19 (#319)
NilPointer-Software/bdfd-wiki: src/bdscript/if.md

# $if

> Returns `true` if provided condition is true, otherwise `false` is returned.

Reviewer: ```suggestion
Executes the following block of code if the provided condition is true.
```
